tools/tools.go

```diff
@@ -15,6 +15,7 @@ import (
 	_ "github.com/gordonklaus/ineffassign"
 	_ "github.com/maxbrunsfeld/counterfeiter/v6"
 	_ "github.com/onsi/ginkgo/v2/ginkgo"
+	_ "github.com/securego/gosec/v2/cmd/gosec"
```

Comment:
It seems this should be in a separate PR to enable its use via the makefile or even GH action.

Reply:
good catch, will remove